src/lib/breadcrumbs/Breadcrumb.svelte

```diff
@@ -0,0 +1,21 @@
+<li
+	class="inline-flex items-center text-sm font-medium text-gray-700 hover:text-primary-600 dark:text-gray-400 dark:hover:text-white"
+>
+	<slot />
+</li>
+
+<style>
+	li::after {
+		content: '\203A';
+		color: darkgray;
+		padding-left: 10px;
+	}
+
+	li:last-child::after {
+		content: '';
+	}
+
+	li:last-child {
+		color: dimgray;
+	}
+</style>
```

Comment:
This should all be doable in tailwind,. for reference: https://tailwindcss.com/docs/content

Reply:
Thank you